test_enclave/build.rs

```diff
@@ -177,7 +209,17 @@ fn build_dynamic_enclave_binary<P: AsRef<Path>>(static_enclave: P) -> PathBuf {
         .arg("--no-undefined")
         .arg("--nostdlib")
         .arg("--start-group")
-        .args(&["--whole-archive", &trts, "--no-whole-archive"])
+        .arg("--whole-archive")
+        .arg(&trts);
+    if let Some(_) = keyfile {
+        if cfg!(feature = "sim") {
+            command.arg("-lsgx_pclsim");
+        } else {
+            command.arg("-lsgx_pcl");
+        }
+    }
```

Comment:
Did this get added in a later commit? I think I just didn't know it existed when I wrote this piece of code...

Reply:
It was there for initial ffi release, https://docs.rs/mc-sgx-core-build/0.1.0/mc_sgx_core_build/fn.sgx_library_suffix.html but it could probably use better documentation